src/marqo/tensor_search/tensor_search.py

```diff
@@ -297,15 +297,15 @@ def _add_documents_unstructured(config: Config, add_docs_params: AddDocsParams,
                     embeddings = [copied[field]["vector"]]
                     # If normalize_embeddings is true and the index version is > 2.13.0, normalize the embeddings.
                     # We have added version specific check here to prevent backwards compatibility issues.
-                    if marqo_index.normalize_embeddings and marqo_index.parsed_marqo_version() > MARQO_CUSTOM_VECTOR_NORMALIZATION_MINIMUM_VERSION:
+                    if marqo_index.normalize_embeddings and marqo_index.parsed_marqo_version() >= MARQO_CUSTOM_VECTOR_NORMALIZATION_MINIMUM_VERSION:
                         try:
                             embeddings = normalize_vector(embeddings)
                         except core_exceptions.ZeroMagnitudeVectorError as e:
                             document_is_valid = False
                             unsuccessful_docs.append(
                                 (i, MarqoAddDocumentsItem(
                                     id=doc_id if doc_id is not None else '',
-                                    error=e.message,
+                                    error=e.__str__(),
```

Comment:
error should have the same content as message for compatibility reasons. (also, it's better to use str(e) instead of magic methods directly)

Reply:
changed back to error = e.message everywhere